code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
I think that's basically what I did. Might be worth making sure you've got a completely clean build (i.e. in apps do `npm run clean` then `npm build`)